src/integrationtest/java/com/ericsson/gerrit/plugins/eiffel/integrationtest/ServiceIntegrationSteps.java

```diff
@@ -0,0 +1,282 @@
+package com.ericsson.gerrit.plugins.eiffel.integrationtest;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.TimeoutException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.apache.http.Header;
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.junit.After;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.ericsson.eiffelcommons.utils.HttpExecutor;
+import com.ericsson.eiffelcommons.utils.HttpRequest;
+import com.ericsson.eiffelcommons.utils.HttpRequest.HttpMethod;
+import com.ericsson.eiffelcommons.utils.ResponseEntity;
+import com.ericsson.gerrit.plugins.eiffel.exceptions.RegexMatchFailedException;
+import com.google.gson.JsonObject;
+import com.rabbitmq.client.AMQP;
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
+import com.rabbitmq.client.GetResponse;
+
+import cucumber.api.java.Before;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+
+public class ServiceIntegrationSteps {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ServiceIntegrationSteps.class);
+
+    private final static String RABBITMQ_HOSTNAME = "localhost";
+    private final static int RABBITMQ_PORT = 5672;
+    private final static String RABBITMQ_USERNAME = "myuser";
+    private final static String RABBITMQ_PASSWORD = "myuser";
+    private final static String RABBITMQ_QUEUENAME = "gerrit-test-queue";
+    private final static String RABBITMQ_EXCHANGE_NAME = "eiffel.poc";
```

Comment:
I think we should remove "poc" word. Name the exchange to "eiffel-exchange" or something.

Reply:
Changed in the next commit.

Reply:
ok